os_info.yaml

```diff
@@ -123,3 +129,9 @@ ubuntu-2204:
     - ppc64le
     - s390x
   type: deb
+ubuntu-2210:
+  version_name: kinetic
+  arch:
+    - amd64
+    - aarch64
```

Comment:
Do we need ppc64le for ubuntu?

Reply:
per commit messages leaving these out due to resource constraints on non-lts ubuntu releases.